metrics/Cargo.toml

```diff
@@ -21,14 +21,23 @@ debug = true
 opt-level = 3
 lto = true
 
+[features]
+default = ["exporters", "recorders"]
+exporters = ["metrics-exporter-log"]
```

Comment:
again here prob can drop all the word metrics from these features?

Reply:
For the features, they need to match the name of the package.  This utilizes Cargo's support for [defining a feature name to include an optional dependency](https://doc.rust-lang.org/cargo/reference/manifest.html#the-features-section).

Reply:
oh im blind :P